src/utils.ts

```diff
@@ -129,6 +129,8 @@ export function getAnimeQueries(name: string): string[] {
 		animeQueries.push(cleanseSeparators(`${title} ${release}`));
 	}
 	if (altTitle) {
+		if (altTitle.toLowerCase() === "season") return animeQueries;
+		if (altTitle.toLowerCase() === "ep") return animeQueries;
```

Comment:
why this change?

Reply:
Some of my anime was parsing these as alternate titles due to weird naming. It would end up searching "season" or "ep". It's easier to reject these here instead of through the regex.

Reply:
can you add a comment to that effect?